profiles/arch/x86/package.use.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Matt Turner <mattst88@gentoo.org> (2021-05-31)
```

Comment:
Just add `~x86`, nobody cares anyway.

Reply:
Fair point wrt keywording and revbump vs masking. Will do